tapchannel/aux_invoice_manager_test.go

```diff
@@ -73,14 +73,37 @@ type mockRfqManager struct {
 	localSellQuotes rfq.SellAcceptMap
 }
 
+// PeerAcceptedBuyQuotes returns buy quotes that were requested by our node and
+// have been accepted by our peers. These quotes are exclusively available to
+// our node for the acquisition of assets.
 func (m *mockRfqManager) PeerAcceptedBuyQuotes() rfq.BuyAcceptMap {
 	return m.peerBuyQuotes
 }
 
+// LocalAcceptedSellQuotes returns sell quotes that were accepted by our node
+// and have been requested by our peers. These quotes are exclusively available
+// to our node for the sale of assets.
 func (m *mockRfqManager) LocalAcceptedSellQuotes() rfq.SellAcceptMap {
 	return m.localSellQuotes
 }
 
+// AssetMatchesSpecifier checks if the provided asset satisfies the provided
+// specifier. If the specifier includes a group key, we will check if the asset
+// belongs to that group.
+func (m *mockRfqManager) AssetMatchesSpecifier(ctx context.Context,
+	specifier asset.Specifier, id asset.ID) (bool, error) {
+
+	switch {
+	case specifier.HasGroupPubKey():
+		return true, nil
```

Comment:
Why do we just return true for group keys? We should expand the unit tests here and earlier in the PR to make sure to matches properly re same group key but diff asset IDs.